recipes/tests.rb

```diff
@@ -303,6 +303,16 @@ module load intelmpi && mpirun --help | grep 'Version 2019 Update 8'
   end
 end
 
+###################
+# EFA - GDR
```

Comment:
What is GDR? could you add the acronym definition?

Reply:
I'm not sure. The GDR acronym is exposed to the user via the `enable_efa_gdr` property.